server/helpers/token.helper.js

```diff
@@ -2,14 +2,12 @@ const jwt = require('jsonwebtoken');
 const { secret, expiresIn } = require('../config/jwt.config');
 
 const verifyToken = async (token) => {
```

Comment:
```suggestion
const verifyToken = token => new Promise((resolve, reject) => {
  jwt.verify(token, secret, (err, authorizedData) => {
    if (err) {
      reject(err);
    }
    resolve(authorizedData);
  });
});
```
Функция не возвращает результат